source/main.c

```diff
@@ -34,10 +34,6 @@ struct shortcut {
   {PAD_BUTTON_Y,     "/y.dol"    },
   {PAD_TRIGGER_Z,    "/z.dol"    },
   {PAD_BUTTON_START, "/start.dol"},
-  {PAD_BUTTON_LEFT,  "/left.dol" },
-  {PAD_BUTTON_RIGHT, "/right.dol"},
-  {PAD_BUTTON_UP,    "/up.dol"   },
-  // Down is reserved for debuging (delaying exit).
```

Comment:
Maybe this should be part of #20 instead?

Reply:
Oops. Yea I did both at the same time and split afterwards. Fixed with a force push.